backend/vite.config.ts

```diff
@@ -0,0 +1,73 @@
+import { defineConfig } from 'vite';
+import { resolve } from 'path';
+
+export default defineConfig({
+  // Disable Vite's dev server for backend
+  build: {
+    outDir: 'dist',
+    lib: {
+      entry: resolve(__dirname, 'src/server.ts'),
+      name: 'SeattleEventsServer',
+      formats: ['cjs'] // CommonJS format for Node.js
```

Comment:
i should figure out how to properly setup a ts project. rn i have no idea why we're using cjs for anything and why editors think we're using node/nodenext instead of ESM for module resolution.